org.eclipse.lsp4j.debug/src/main/java/org/eclipse/lsp4j/debug/DebugProtocol.xtend

```diff
@@ -2627,6 +2627,15 @@ class Capabilities {
 	 * Since 1.51
 	 */
 	Boolean supportsSingleThreadExecutionRequests;
+	
+	/**
+  	 * Client supports the `startDebugging` request.
+  	 * <p>
+	 * This is an optional property.
+	 * <p>
+	 * Since 1.58
```

Comment:
I think it is since 1.59 - but it is missing from the changelog: https://github.com/microsoft/debug-adapter-protocol/issues/378